core/migrations/11_deploy_tokenized_derivative_creator.js

```diff
@@ -3,32 +3,45 @@ const TokenizedDerivativeCreator = artifacts.require("TokenizedDerivativeCreator
 const TokenizedDerivativeUtils = artifacts.require("TokenizedDerivativeUtils");
 const LeveragedReturnCalculator = artifacts.require("LeveragedReturnCalculator");
 const AddressWhitelist = artifacts.require("AddressWhitelist");
-const {
-  getKeysForNetwork,
-  deployAndGet,
-  enableControllableTiming,
-  addToTdr
-} = require("../../common/MigrationUtils.js");
+const { getKeysForNetwork, deploy, enableControllableTiming, addToTdr } = require("../../common/MigrationUtils.js");
+
+const ethAddress = "0x0000000000000000000000000000000000000000";
 
 module.exports = async function(deployer, network, accounts) {
   const keys = getKeysForNetwork(network, accounts);
   const controllableTiming = enableControllableTiming(network);
 
   // Deploy whitelists.
-  const sponsorWhitelist = await deployAndGet(deployer, AddressWhitelist, { from: keys.sponsorWhitelist });
-  const returnCalculatorWhitelist = await deployAndGet(deployer, AddressWhitelist, {
-    from: keys.returnCalculatorWhitelist
-  });
-  const marginCurrencyWhitelist = await deployAndGet(deployer, AddressWhitelist, {
-    from: keys.marginCurrencyWhitelist
-  });
+  const { contract: sponsorWhitelist, didDeploy: deployedSponsorWhitelist } = await deploy(
```

Comment:
Looks like the `deployed*` consts aren't being used for anything?

Reply:
Nice catch - you're right, that was from an earlier iteration of the code that I forgot to delete.